src/cmd/services/m3dbnode/config/bootstrap.go

```diff
@@ -43,14 +43,21 @@ import (
 var (
 	// defaultNumProcessorsPerCPU is the default number of processors per CPU.
 	defaultNumProcessorsPerCPU = 0.125
+
+	// order in which bootstrappers are run
+	// (run in ascending order of precedence)
+	orderedBootstrappers = []string{
+		uninitialized.UninitializedTopologyBootstrapperName,
+		// Peers and commitlog must come after uninitialized topology bootrapping.
+		peers.PeersBootstrapperName,
+		commitlog.CommitLogBootstrapperName,
+		// Filesystem bootstrapping must be last.
```

Comment:
So fs bootstrapper actually gets run first. This config is correct though I think since when we construct each bootstrapper we pass in the previous bootstrapper as the bootstrapper to call next. 

Maybe make a note of this visually reversed run order in the comments above?

Reply:
Ah ok yeah the original way in which the config order that is passed in is then reversed was confusing which is why I then ordered them this way. But I'll make explicit what's going on here in comments.